addons/website/views/website_footer_templates.xml

```diff
@@ -0,0 +1,166 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!-- Footer templates avaible in the customize theme menu-->
+    <!-- Logo - About us (below)-->
```

Comment:
I won't create a new file for this, put this in website_templates next to footers